rtree.go

```diff
@@ -696,6 +696,19 @@ func sortEntries(p Point, entries []entry) ([]entry, []float64) {
 	return sorted, dists
 }
 
+func sortPreallocEntries(p Point, entries, sorted []entry, dists []float64) ([]entry, []float64) {
```

Comment:
To reduce code duplication you could change `sortEntries` to
```golang
func sortEntries(p Point, entries []entry) ([]entry, []float64) {
    sorted := make([]entry, len(entries))
    dists := make([]float64, len(entries))
    return sortPreallocEntries(p, entries, sorted, dists)
}
```

Reply:
Nice! Mache ich noch.